tests/integration/targets/locale_gen/tasks/main.yml

```diff
@@ -10,7 +10,8 @@
 
 - name: Bail out if not supported
   ansible.builtin.meta: end_play
-  when: ansible_distribution not in ('Ubuntu', 'Debian')
+  when:
+    - "ansible_distribution == 'MacOSX'"
```

Comment:
macOS is already excluded (together with FreeBSD and AIX) by `aliases`, so you can also simply remove the task.

You need to fix the tests for all the new platforms the PR is running them on, though.